src/services/stac/index.tsx

```diff
@@ -99,6 +99,103 @@ export const getStacItems = async (
   };
 };
 
+export const fetchFavouritedItems = async (collectionId: string): Promise<string[]> => {
+  const workspace = (await getActiveWorkspace()) || 'james-hinton';
+  const url = `${import.meta.env.VITE_STAC_ENDPOINT}/catalogs/user-datasets/${workspace}/saved-data/collections/${collectionId}/items?limit=99999`;
+
+  try {
+    const response = await fetch(url, {
```

Comment:
I am getting 403 forbidden errors when I try to make this request is that to be expected? Is it because I do not have a workspace with the name james-hinton? The UI itself is responding how I would expect, you can visually see when you have favourited  / un-favourited an item
<img width="401" alt="image" src="https://github.com/user-attachments/assets/2cb3b02f-52aa-4c8e-b001-7f466519b053">